watertap/data/techno_economic/co2_addition.yaml

```diff
@@ -0,0 +1,20 @@
+default:
+  energy_electric_flow_vol_inlet:
+    value: 0.0
+    units: kWh/m^3
+  specific_capital_cost_flow_vol_inlet:
+    base_fixed_cap_cost:
```

Comment:
I actually labeled `base_fixed_cap_cost` as `cap_basis_par` in other merged PRs, with the thought that it might be easier to use that same name (just a different cost_function) when @andrewlee94 brings costing in. However, I'm not sure how useful that'd be.

Reply:
That is fine with me. I wasn't sure so was just using the variables in WT3.

Reply:
```suggestion
    cap_basis_par:
```

Reply:
I used a different name myself in the costing - for now I suggest we not touch these until we can review the changes in the costing code and settle on a convention.